### PR TITLE
Add info about Node app in Oryx's build manifest file

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: src/BuildScriptGenerator/Node/NodeManifestFilePropertyKeys.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace Microsoft.Oryx.BuildScriptGenerator.Node
+{
+    public class NodeManifestFilePropertyKeys
+    {
+        /// <summary>
+        /// The name of the property whose value indicates the location of the output directory of a Node app.
+        /// For example, 'dist' or 'public' folder.
+        /// </summary>
+        public const string OutputDirPath = nameof(OutputDirPath);
```
Comment (gpcastro) [inline src/BuildScriptGenerator/Node/NodeManifestFilePropertyKeys.cs]: is it really the output, or just some specific component like static content?

File: src/BuildScriptGenerator/Node/NodePlatform.cs [hunk 2]
```diff
@@ -168,6 +170,53 @@ public BuildScriptSnippet GenerateBashBuildScriptSnippet(BuildScriptGeneratorCon
             };
         }
 
+        private void GetAppOutputDirPath(dynamic packageJson, Dictionary<string, string> buildProperties)
```
Comment (gpcastro) [inline src/BuildScriptGenerator/Node/NodePlatform.cs]: nit: this method is actually getting the value but also setting it in the build properties. Should we split this in two? Or rename it as it's not called `Get`?

File: src/BuildScriptGenerator/Node/NodeManifestFilePropertyKeys.cs (hunk 1 above)
Comment (kichalla) [reply src/BuildScriptGenerator/Node/NodeManifestFilePropertyKeys.cs]: As I mentioned in scrum today, this has been the typical naming pattern by different frameworks, so I kept it this way.

File: src/BuildScriptGenerator/Node/NodePlatform.cs (hunk 2 above)
Comment (kichalla) [reply src/BuildScriptGenerator/Node/NodePlatform.cs]: Sure I agree. Will change it in a different PR.

**Pull request merged**